src/main/java/org/gelecekbilimde/scienceplatform/auth/service/RoleApplicationService.java

```diff
@@ -0,0 +1,17 @@
+package org.gelecekbilimde.scienceplatform.auth.service;
+
+import org.gelecekbilimde.scienceplatform.auth.model.RoleApplication;
+import org.gelecekbilimde.scienceplatform.auth.model.request.RoleChangeRequestsFilter;
+import org.springframework.data.domain.Page;
+
+import java.util.List;
+
+public interface RoleApplicationService {
+
+	Page<RoleApplication> findAll(List<RoleChangeRequestsFilter> filters, int page, int size);
+
+	void approve(String id);
+
+	void reject(String id);
```

Comment:
@Coderabbitai We use UUIDs as `String` throughout the application. If we want to make changes to the IDs in the future, we have made them flexible so that the application is not affected in general. For this reason, don't suggest it again in the next suggestions.